backoffice/modules/catalog/components/CustomInput.tsx

```diff
@@ -0,0 +1,94 @@
+import { ProductVariation } from '@catalogModels/ProductVariation';
+import React, { useState, useEffect } from 'react';
+import { Form, Badge } from 'react-bootstrap';
+import { toast } from 'react-toastify';
+
+type CustomInputProps = {
+  defaultValue?: string;
+  onChange?: (value: string[]) => void;
+  productVariations?: ProductVariation[];
+};
+
+const CustomInput: React.FC<CustomInputProps> = ({ defaultValue, onChange, productVariations }) => {
```

Comment:
Hi, as far as I understand, this is a customized input specifically for creating option values, right? In that case, the name should reflect its purpose more closely, as the current name feels a bit too general.